boundary_layer/registry/registry.py

```diff
@@ -197,6 +197,10 @@ def load_configs(self, config_paths):
         for path in config_paths:
             logger.debug('Loading configs from path %s', path)
             for filename in os.listdir(path):
+                # Skip Vim swap files (for local development)
+                if os.path.splitext(filename)[1] in ['.swp', '.swo']:
```

Comment:
Minor but could you also add a check for whether the file name starts with `.` ?